<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MULTIPLAYER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SINGLEPLAYER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Gorączka, kaszel, zmęczenie, ból gardła, a nawet śmierć!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Oto nadchodzi niebezpieczny wirus COVID-19! &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Zarażenie drogą kropelkową jest bardzo realne, &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;zatem zachowaj higienę i trzymaj się z daleka od zainfekowanych. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Czy uda Ci się uchronić przed śmiertelnym zagrożeniem? &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Przetrwaj, zbierając zapasy makaronu, papieru toaletowego&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;i maseczek ochronnych oraz unikaj kontaktu z policjantami &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;i &lt;b&gt;wirusem&lt;/b&gt; tak długo jak to możliwe!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="strzalki">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>←↓→↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asdw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ASDW</string>
    </property>
   </widget>
   <widget class="QLabel" name="klawisz">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>10</y>
      <width>361</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bauhaus 93</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WYBIERZ GRACZA I KLAWISZE:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cofnij">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>COFNIJ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>GRACZ 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>60</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>GRACZ 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuGRA">
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>GRA</string>
    </property>
    <widget class="QMenu" name="menuNowa_gra">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Nowa gra</string>
     </property>
     <widget class="QMenu" name="menuSingle_playey">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Single playey</string>
      </property>
      <addaction name="actionASDW"/>
      <addaction name="action"/>
     </widget>
     <addaction name="menuSingle_playey"/>
     <addaction name="actionMultiplayer"/>
    </widget>
    <addaction name="menuNowa_gra"/>
    <addaction name="actionInstrukcja"/>
    <addaction name="actionZako_cz"/>
   </widget>
   <addaction name="menuGRA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMultiplayer">
   <property name="text">
    <string>Multiplayer</string>
   </property>
  </action>
  <action name="actionASDW">
   <property name="text">
    <string>ASDW</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>←↓→↑</string>
   </property>
  </action>
  <action name="actionInstrukcja">
   <property name="text">
    <string>Instrukcja</string>
   </property>
  </action>
  <action name="actionZako_cz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
